examples/python/structure_from_motion/main.py

```diff
@@ -23,6 +24,37 @@
 # When dataset filtering is turned on, drop views with less than this many valid points.
 FILTER_MIN_VISIBLE: Final = 500
 
+DESCRIPTION = """
+# Sparse Reconstruction by COLMAP
+
+This example was generated from the output of a sparse reconstruction
+done with COLMAP.
+
+[COLMAP](https://colmap.github.io/index.html) is a general-purpose
+Structure-from-Motion (SfM) and Multi-View Stereo (MVS) pipeline
+with a graphical and command-line interface.
+
+In this example a short video clip has been processed offline by the
+COLMAP pipeline, and we use Rerun to visualize the individual
+camera frames, estimated camera poses, and resulting point clouds over time.
+
+## How it was made
+The full source code for this example is available on Github
+[here](https://github.com/rerun-io/rerun/blob/latest/examples/python/structure_from_motion/main.py).
+
+### Colored 3D Points
+The colored 3D points were added to the scene by logging the
+[rr.Points3D archetype](https://www.rerun.io/docs/reference/data_types/point3d)
```

Comment:
Would have loved to be able to write
```
[`rr.Points3D` archetype](https://www.rerun.io/docs/reference/data_types/point3d)
```
here instead to make the `rr.Points3D` be formatted as both code and a link but not super important.